ethcore/res/ethereum/ethercore.json

```diff
@@ -0,0 +1,211 @@
+		"author": "0x0000000000000000000000000000000000000000",
+		"timestamp": "0x5e0be100",
+		"parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+		"extraData": "0x0000000000000000000000000000000000000000000000000000000000000000",
+		"gasLimit": "0x7a1200"
+	},
+	"nodes": [
+		"enode://dfcfd268e7d4631cec5a94d7eeb5981a2ed3f30235c774e1bf0a4e843672405e5a1c93502e1631edf108b6f1ea8701fb97e6d892eb5e3775c70dc53b2f773b9f@207.148.105.65:30303",
+		"enode://6c67afac3a018ee5641b0aba996f180fbb718aa7857174cad568b00440a43cccb1845124cd1cbe43c7ce1e95d597a3d20d175f0bf8494d875e83ec15e0f42cb4@207.148.105.65:30503"
+	],
+	"accounts": {
+		"0x0000000000000000000000000000000000000001":{
+			"builtin":{
+				"name":"ecrecover",
+				"pricing":{
+					"linear":{
+						"base":3000,
+						"word":0
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000002":{
+			"builtin":{
+				"name":"sha256",
+				"pricing":{
+					"linear":{
+						"base":60,
+						"word":12
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000003":{
+			"builtin":{
+				"name":"ripemd160",
+				"pricing":{
+					"linear":{
+						"base":600,
+						"word":120
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000004":{
+			"builtin":{
+				"name":"identity",
+				"pricing":{
+					"linear":{
+						"base":15,
+						"word":3
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000005":{
+			"builtin":{
+				"activate_at":"0x0",
+				"name":"modexp",
+				"pricing":{
+					"modexp":{
+						"divisor":20
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000006":{
+			"builtin":{
+				"name":"alt_bn128_add",
+				"pricing":{
+					"0x0":{
+						"price":{
+							"alt_bn128_const_operations":{
+								"price":500
+							}
+						}
+					},
+					"0xbe10b":{
+						"info":"EIP 1108 transition",
+						"price":{
+							"alt_bn128_const_operations":{
+								"price":150
+							}
+						}
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000007":{
+			"builtin":{
+				"name":"alt_bn128_mul",
+				"pricing":{
+					"0x0":{
+						"price":{
+							"alt_bn128_const_operations":{
+								"price":40000
+							}
+						}
+					},
+					"0xbe10b":{
+						"info":"EIP 1108 transition",
+						"price":{
+							"alt_bn128_const_operations":{
+								"price":6000
+							}
+						}
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000008":{
+			"builtin":{
+				"name":"alt_bn128_pairing",
+				"pricing":{
+					"0x0":{
+						"price":{
+							"alt_bn128_pairing":{
+								"base":100000,
+								"pair":80000
+							}
+						}
+					},
+					"0xbe10b":{
+						"info":"EIP 1108 transition",
+						"price":{
+							"alt_bn128_pairing":{
+								"base":45000,
+								"pair":34000
+							}
+						}
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000009": {
+			"builtin": {
+				"name": "blake2_f",
+				"activate_at": "0x0",
```

Comment:
Rationale why `blake2_f` is activated at 0 and `alt_bn128_xx` cost reductions at `0xbe10b`?

Do you follow the EIPs activated on Istanbul or something else?

Reply:
Nice catch, thank you!

Reply:
ah, I wonder if there is a tool to warn you about potential errors of copy-pasting code :)